invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -4,85 +4,289 @@
       "date_detection": false,
       "numeric_detection": false,
       "properties": {
-        "_access": {
+        "_bucket": {
+          "enabled": false
+        },
+        "_recid": {
+          "type": "keyword"
+        },
+        "_conceptrecid": {
+          "type": "keyword"
+        },
+        "_visibility": {
+          "type": "keyword"
+        },
+        "_visibility_files": {
+          "type": "keyword"
+        },
+        "_embargo_date": {
+          "type": "date"
+        },
+        "_access_condition": {
           "type": "object",
           "properties": {
-            "metadata_restricted": {
-              "type": "boolean"
+            "condition": {
+              "type": "keyword"
             },
-            "files_restricted": {
-              "type": "boolean"
+            "link_validity": {
+              "type": "keyword"
             }
           }
         },
-        "_bucket": {
-          "enabled": false
-        },
-        "access_right": {
+        "_contact": {
```

Comment:
The jsonschema one doesn't start with a `_`. This one shouldn't I believe.

Reply:
The JSONSchema will validate the field name, then marshmallow will move it to underscore one. This three (contact, community, and embargo date) and fields that are well into metadata of the record in the sense that they provide useful information, but they are in reality internal fields (so treated with `_`).